cartographer/mapping/sparse_pose_graph.h

```diff
@@ -68,6 +68,12 @@ class SparsePoseGraph {
     transform::Rigid3d pose;
   };
 
+  struct InitialTrajectoryPose {
+    int to_trajectory_id;
+    transform::Rigid3d relative_pose;
+    common::Time time;
+  };
+
```

Comment:
Remove and use mapping::proto::InitialTrajectoryPose?

Reply:
I don't think we want to stick protos into the map.